### PR TITLE
bump py-bbp-workflow

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py [hunk 1]
```diff
@@ -10,9 +10,9 @@ class PyLuigiTools(PythonPackage):
     '''Tools to work with luigi.'''
 
     homepage = 'https://github.com/BlueBrain/luigi-tools'
-    url      = 'https://pypi.io/packages/source/l/luigi-tools/luigi-tools-0.0.15.tar.gz'
+    url      = 'https://pypi.io/packages/source/l/luigi-tools/luigi-tools-0.0.16.tar.gz'
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py]:
```suggestion
    pypi     = 'luigi-tools/luigi-tools-0.0.16.tar.gz'
```
Should work, too. And the version number does not need to be kept in sync with the defined ones :)

**Review by matz-e — DISMISSED** [review]

File: bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py (hunk 1 above)
Comment (genric) [reply bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py]: yeah, just as we don't accumulate multiple versions, I find it nicer to keep them in sync :)

**Review by matz-e — APPROVED** [review]

File: bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py (hunk 1 above)
Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py]: Fair enough, less confusion probably. Now that I've disentangled repositories, I don't expect merge conflicts from it anymore, either.

**Comment by genric** [comment]
@matz-e, not sure if this is a permissions issue you were looking for :) but `ci/gitlab/bbpgitlab.epfl.ch` is still pending here in the checks list, I also didn't see anything appear in gitlab pipelines... :(

**Comment by matz-e** [comment]
> @matz-e, not sure if this is a permissions issue you were looking for :) but `ci/gitlab/bbpgitlab.epfl.ch` is still pending here in the checks list, I also didn't see anything appear in gitlab pipelines... :(

Indeed — yes, that's the permissions issue. Now I'm completely unaware of how to fix this, but let me try and dig in.

**Comment by matz-e** [comment]
Actually, @genric, you're wanting to merge from a branch that is not in this repo. I'd like to keep that disabled, as I can't whitelist people like with Jenkins.

Could you see if you can write to this repo and create a branch here, please?

**Comment by genric** [comment]
Yeah, that was my guess as well :+1:  I'll try with branch here...

**Comment by genric** [comment]
Closed in favor of #1439

**Pull request closed without merging**